front/src/Phaser/Login/EntryScene.ts

```diff
@@ -24,41 +29,50 @@ export class EntryScene extends Scene {
         // Note: arcade.png from the Phaser 3 examples at: https://github.com/photonstorm/phaser3-examples/tree/master/public/assets/fonts/bitmap
         this.load.bitmapFont(ReconnectingTextures.mainFont, "resources/fonts/arcade.png", "resources/fonts/arcade.xml");
         this.load.spritesheet("cat", "resources/characters/pipoya/Cat 01-1.png", { frameWidth: 32, frameHeight: 32 });
+        this.translator.loadCurrentLanguageFile(this.load);
     }
 
     create() {
-        gameManager
-            .init(this.scene)
-            .then((nextSceneName) => {
-                // Let's rescale before starting the game
-                // We can do it at this stage.
-                waScaleManager.applyNewSize();
-                this.scene.start(nextSceneName);
+        this.translator
+            .loadCurrentLanguageObject(this.cache)
+            .catch((e: unknown) => {
+                console.error("Error during language loading!", e);
+                throw e;
```

Comment:
Does Phaser continue running after create a throw?

We should probably do a test here.